spec/unit/services/action_service_spec.rb

```diff
@@ -114,20 +115,19 @@
       end
 
       describe 'getCookiesProp' do
-
-        let(:cookies) { { 'name' => 'John' } }
         let(:script) { "return getCookiesProp('name');" }
-
-        it { is_expected.to eq 'John' }
-
+        before do
+          expect(cookie_service).to receive(:get).with('name').and_return('John')
+        end
+        it { is_expected.to eq('John') }
       end
 
-      describe 'sendCookiesProp' do
-
-        let(:script) { "return setCookiesProp('done', true);" }
-
-        it { subject; expect(cookies['done']).to eq 'true' }
-
+      describe 'setCookiesProp' do
+        let(:script) { "return setCookiesProp('done', {'value': true});" }
+        before do
+          expect(cookie_service).to receive(:set).with('done', {'value' => true})
```

Comment:
it'd better to move this statement within its own test

Reply:
If you have some minute, I have a question on this

Reply:
I think I got it, tell me if it's good.
I am newbie in ruby testing ;)